packages/tasks/src/model-libraries.ts

```diff
@@ -298,6 +298,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "mindspore",
 		repoUrl: "https://github.com/mindspore-ai/mindspore",
 	},
+	mamba_ssm: {
+		prettyLabel: "MambaSSM",
+		repoName: "MambaSSM",
+		repoUrl: "https://github.com/state-spaces/mamba",
+		filter: false,
+		countDownloads: `path:"config.json"`,
```

Comment:
No needed as this is already one of the files used to count downloads

Reply:
addressed in https://github.com/huggingface/huggingface.js/pull/802/commits/efaf43986ac117fb048848d30ea705f706e60aac